sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Sha1Helper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    internal static class Sha1Helper
```

Comment:
Related to my comment above, we shouldn't support SHA1, so this class should be removed. The only reason that we are able to still use SHA1 certificate thumbprints is because customers can use them, despite Microsoft not being able to. This came up in a Crypto Board discussion recently. This would likely draw serious ire.